src/Illuminate/Database/Migrations/Migrator.php

```diff
@@ -17,6 +18,13 @@
 
 class Migrator
 {
+    /**
+     * The container implementation.
+     *
+     * @var \Illuminate\Contracts\Container\Container|null
+     */
+    protected $container;
```

Comment:
Please order these properties the same order as the constructor.

Reply:
Then I need to reorder all the properties. They are not ordered right now.

Reply:
Done